### DISCUSSION TITLE
希望改变一些组件的动画效果

### DISCUSSION THREAD

**Discussion by xmsz-stu** [DouyinFE/semi-design#2131] [Ideas]
现在有些组件的动画都是在有点太生硬了。虽然说动画没啥大作用，但是有的话也算个锦上添花的事情。

比如，Checkbox/Radio
想象中，应该是点击的时候，背景从外部往内部填充，然后勾有个小特效
![CleanShot 2024-03-18 at 15 14 42](https://github.com/DouyinFE/semi-design/assets/139225486/6eaaddfb-48eb-44a9-bd92-b67126a82c75)
当然偷懒点，可以是这种只有背景动画，https://opensource.adobe.com/spectrum-web-components/components/checkbox/，或者只有勾出现的动画也行。

比如，Tabs
现在一个问题是点击按钮的时候会先从灰色变成黑色，再从黑色变为粗体，整体感觉就像闪了一下的感觉。第二个问题是底部的指示条出现的太突兀。
想象中，先解决一下闪烁的效果
然后是底部的标识条，可以有种跟随的效果

**Comment by SudoUserReal** [comment]
已有的动画可以通过我们的  主题平台 https://semi.design/dsm 来修改渐变函数和持续时间以及延迟。加新的动画的话，只能自食其力自己动手了。。欢迎给我们提PR

**Comment by SudoUserReal** [comment]
但是涉及到设计观感的修改，提PR 前建议先建立Issue 和我们讨论

**Reply by xmsz-stu** [reply]
ok
